averages.js

```diff
@@ -0,0 +1,31 @@
+function total(arr) {
+  var tote = arr.reduce(function (sum, value) {
+    return sum + value;
+  }, 0);
+  return tote;
+}
+
+function mean(arr) {
+  var count = arr.length;
+  var tote = total(arr);
+  return tote / count;
+}
+
+function median(arr) {
+  // cannot get this fucker to work!
+  var midPoint = Math.floor(arr.length);
+  var med = arr.sort(function (a,b) {
```

Comment:
Close. Remember that the array could have an odd or even number of elements and we need to treat those differently to get the median